### ISSUE TITLE
Break out script types into separate repos in the org?

### ISSUE THREAD

**Issue by therealscottcarlow** [MSP-Zone-Unofficial/Scripts#4]
It may be better for organization, considering the possibly vast amount of sub-categories within each language, to break things out into separate repos. Thoughts?

**Comment by NRClay** [comment]
I was thinking of something similar. Is there a way to have a over arching project?

**Comment by NRClay** [comment]
Also since we are talking about organization, should we break the sections into script components (similar to the one liners I have added) and actual complex scripts?

**Comment by therealscottcarlow** [comment]
Well, we have the Org itself, and that can contain unlimited repos under its umprella. I gave org members the ability to create repos and to push (no more need for PR's unless you're and outside contributor), so that makes things a bit easier on the collab front.

**Comment by therealscottcarlow** [comment]
I don't think we need to break the one-liners away from the complex scripts. They both have their purpose, and they are in fact both scripts.
